CHANGELOG.md

```diff
@@ -24,6 +24,12 @@ and this project adheres to the versioning scheme outlined in the [README.md](RE
 - Added config setting `burnchain.wallet_name` which addresses blank wallets no 
   longer being created by default in recent bitcoin versions. Fixes issue #3596
 
+## [2.4.0.0.1]
```

Comment:
Since this is framed as 2.4.0.0.1, should it be opened against master?

Reply:
good point - i'll remove the version section here and add to the correct changelog location